application-xpoll-ui/src/main/resources/XPoll/XPollJSExtension.xml

```diff
@@ -151,6 +151,23 @@
         }
       });
     });
+    $('.xpollSelect').change(function() {
+      var changedElement = $(this);
+      $('.xpollSelect').each(function() {
+        if ($(this).data('proposal') != changedElement.data('proposal') &amp;&amp; $(this).val() ==
+        changedElement.val())
```

Comment:
Bad formatting. Moreover, the best practice is to always use a block ``{}`` with IF, as Lavinia pointed out below. If you rely only o indentation then the next person that adds a new line "inside" the IF will have a unpleasant surprise.